webapp/channels/src/components/signup/signup.tsx

```diff
@@ -893,13 +892,27 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
                             )}
                             {enableSignUpWithEmail && !serverError && (
                                 <p className='signup-body-card-agreement'>
-                                    <FormattedMarkdownMessage
-                                        id='create_team.agreement'
-                                        defaultMessage='By proceeding to create your account and use {siteName}, you agree to our [Terms of Use]({TermsOfServiceLink}) and [Privacy Policy]({PrivacyPolicyLink}). If you do not agree, you cannot use {siteName}.'
+                                    <FormattedMessage
+                                        id='signup.agreement'
+                                        defaultMessage='By proceeding to create your account and use {siteName}, you agree to our <a1>Terms of Use</a1> and <a2>Privacy Policy</a2>.  If you do not agree, you cannot use {siteName}.'
```

Comment:
Could you name these `termsOfUseLink` and `privacyPolicyLink` instead of `a1` and `a2` so that they have more descriptive names?

Reply:
changed